src/main/java/seedu/address/logic/parser/profile/DeleteProfileCommandParser.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser.profile;
+
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_OPTION;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.profile.DeleteProfileCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.profile.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteProfileCommand object
+ */
+public class DeleteProfileCommandParser implements Parser<DeleteProfileCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteCommand
+     * and returns a DeleteCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteProfileCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_OPTION);
+        String indexInput = argMultimap.getOptionArgs();
+        if (indexInput.equals("")) {
```

Comment:
Should this be `indexInput.isEmpty()` instead, which might make the code easier to understand?

Reply:
Thanks for catching the errors!